# Define the goal
geometry_msgs/Pose target_object_pose
---
# Define the result
---
# Define the feedback
